src/main/java/Chatbot.java

```diff
@@ -0,0 +1,208 @@
+import java.sql.Array;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Chatbot {
+    private Scanner scanner = new Scanner(System.in);
+    private String currentInput;
+    private ArrayList<Task> list;
+
+
+    public Chatbot() {
+        this.currentInput = "";
+        this.list = new ArrayList<Task>();
+    }
+
+    public void start() {
+        greetings();
+        this.currentInput = scanner.nextLine();
+        while (!this.currentInput.equals("bye")) {
+            parseInput();
+            this.currentInput = scanner.nextLine();
+        }
+        exit();
+    }
+
+    private boolean isNumeric(String str) {
+        try {
+            Double.parseDouble(str);
+            return true;
+        } catch(NumberFormatException e){
+            return false;
+        }
+    }
+
+    private void parseInput() {
+        if (this.currentInput.equals("list")) {
+            outputList();
+            return;
+        }
+
+        String instruction[] = this.currentInput.split(" ", 2); // It only splits the first " "
+        if (instruction[0].equals("mark")) {
+            this.mark(instruction);
+            return;
+        }
+
+        if (instruction[0].equals("unmark")){
+            this.unmark(instruction);
+            return;
+        }
+
+        if (instruction[0].equals("delete")) {
+            this.delete(instruction);
```

Comment:
Consider removing the 'this' reference since there is inconsistent styling with addToDo among other instructions
```suggestion
            delete(instruction);
```